src/app/content/components/Attribution.spec.tsx

```diff
@@ -2,29 +2,23 @@ import cloneDeep from 'lodash/fp/cloneDeep';
 import createTestStore from '../../../test/createTestStore';
 import { book, page, shortPage } from '../../../test/mocks/archiveLoader';
 import { mockCmsBook } from '../../../test/mocks/osWebLoader';
-import { resetModules } from '../../../test/utils';
+import { reactAndFriends, resetModules } from '../../../test/utils';
 import { AppState, Store } from '../../types';
 import * as actions from '../actions';
 import { initialState } from '../reducer';
 import { formatBookData } from '../utils';
 
 describe('Attribution', () => {
-  let React: any; // tslint:disable-line:variable-name
-  let ReactDOM: any; // tslint:disable-line:variable-name
-  let renderer: any;
-  let Provider: any; // tslint:disable-line:variable-name
-  let renderToDom: any;
-  let MessageProvider = require('../../MessageProvider').default; // tslint:disable-line:variable-name
+  let React: ReturnType<typeof reactAndFriends>['React']; // tslint:disable-line:variable-name
+  let ReactDOM: ReturnType<typeof reactAndFriends>['ReactDOM']; // tslint:disable-line:variable-name
+  let renderer: ReturnType<typeof reactAndFriends>['renderer'];
+  let renderToDom: ReturnType<typeof reactAndFriends>['renderToDom'];
+  let TestContainer: ReturnType<typeof reactAndFriends>['TestContainer']; // tslint:disable-line:variable-name
 
   beforeEach(() => {
     jest.resetAllMocks();
     resetModules();
-    React = require('react');
-    ReactDOM = require('react-dom');
-    Provider = require('react-redux').Provider;
-    renderer = require('react-test-renderer');
-    renderToDom = require('../../../test/reactutils').renderToDom;
-    MessageProvider = require('../../MessageProvider').default;
+    ({React, ReactDOM, renderer, renderToDom, TestContainer} = reactAndFriends());
```

Comment:
@TomWoodward based on some reading I thought something like this in the second `beforeEach` would work but it's having no effect:
```
    const mockScrollTo = scrollTo = jest.fn();
      jest.isolateModules(() => {
        jest.doMock('scroll-to-element', () => mockScrollTo);
        scrollToElem = require('scroll-to-element').default;
      });
```

Reply:
https://stackoverflow.com/questions/45571816/reset-single-module-with-jest

Reply:
the issue has to do with the files you're testing, it might help to put `isolateModules` around where you are requiring `Attribution` but putting it around the mock will not impact the stuff required by the code you're testing.

when i was looking at this i just moved the `scroll-to-element` mock up the logic until it started working, which would be an acceptable fix even though it would put it in scope of more tests, whatever.